equipmanager.lic

```diff
@@ -395,4 +397,82 @@ class EquipmentManager
       stow_helper("stow my #{weapon_name}", weapon_name, 'You put', 'You should unload', 'You easily strap', 'You secure your')
     end
   end
+
+  # Given an item and a description (ajd + noun or just noun)
+  # then returns true if the item matches, else false.
+  def item_matches_desc?(item, desc)
+    return false unless item && desc
+    check_item_matches_regex_pr4484?(item, desc)
+    return (item.short_regex =~ desc) != nil
```

Comment:
Line 405: we check if their yaml config matches the item's adj/noun exactly and warn otherwise in preparation for https://github.com/rpherbig/dr-scripts/pull/4484/files#diff-3cc8e7e4d7960e8338d3d8b4201e6c398747480769cfc458fb288bc1eb2d5b57R322

Line 406: the original comparison logic is still in place